playbooks/roles/topbeat/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+topbeat_app_dir: "{{ COMMON_APP_DIR }}/topbeat"
```

Comment:
I wonder if these two roles, `topbeat` and `filebeat` are close enough
to be abstracted and combined into a single role. The standout
difference I see is that `topbeat` also installs a Python package; I
think we can configure through that though.

I'm imagining something like:
```yaml
---
# topbeat.yml
# playbooks/roles/logstash-beats/defaults/main.yml
# *_app_dir can be intuited from name
LOGSTASH_BEATS_NAME: 'topbeat'
LOGSTASH_BEATS_VERSION: '1.0.0'
logstash_beats_download_filename: "{{ LOGSTASH_BEATS_NAME }}_{{ logstash_beats_version }}_amd64.deb"
logstash_beats_download_url_base: "https://download.elastic.co/beats/{{ LOGSTASH_BEATS_NAME }}/{{ logstash_beats_download_filename }}"
logstash_beats_pip_packages:
    - httplib2
```

```yaml
---
# playbooks/roles/logstash-beats/defaults/filebeat.yml
LOGSTASH_BEATS_NAME: 'filebeat'
LOGSTASH_BEATS_VERSION: '1.2.3'
logstash_beats_pip_packages: []
# everything else is inherited from above
```

Pros, cons?